tests/test_client.py

```
@@ -1,112 +1,113 @@
-        
-        with patch.object(stake_client, '_request', return_value=mock_response) as mock_request:
+
+        with patch.object(
+            stake_client, "_request", return_value=mock_response
+        ) as mock_request:
             await stake_client.get_casino_games(category="slots")
-            
+
             # Verify the request was made with correct parameters
             mock_request.assert_called_once()
             args, kwargs = mock_request.call_args
```

Ruff (0.16.2)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)